solr/api/src/java/org/apache/solr/client/api/model/Constants.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.solr.client.api.model;
+
+public class Constants {
+
+  private Constants() {
+    /* Private ctor prevents instantiation */
+  }
+
+  /** A parameter to specify the name of the backup repository to be used. */
+  public static final String BACKUP_REPOSITORY = "repository";
```

Comment:
[0] The additions here are a real shame, as they're duplicates of constants that already exist in SolrJ.

Should we move some of the solrj "constants" files (i.e. CollectionAdminParams, CommonParams) into 'api' to avoid the duplication?  (To handle backcompat, in 9.x we could have a deprecated copy of each of these constants files in solrj that extends its 'api' module counterpart)